install/vexxhost-installation-guide.md

```diff
@@ -0,0 +1,16 @@
+# Install Kata Containers on VEXXHOST
+
+Kata Containers on VEXXHOST use nested virtualization to provide an identical
+installation experience to Kata on your preferred Linux distribution.
+
+This guide assumes you have an OpenStack public cloud account set up and tools
+to remote into your virtual machine (SSH).  
+
+## Create a new virtual machine with nesting support
+
+All regions support nested virtualization using the V2 flavors (those prefixed
+with v2).  The recommended for container workloads is `v2-highcpu` range.
```

Comment:
Second sentence is awkward for me.  Should it be "The recommendation for" or, "The recommended machine for" ?